protos/camera/camera.proto

```diff
@@ -17,10 +17,6 @@ option java_outer_classname = "CameraProto";
  * `select_camera`.
  */
 service CameraService {
-    /*
-     * Prepare the camera plugin (e.g. download the camera definition, etc).
-     */
-    rpc Prepare(PrepareRequest) returns(PrepareResponse) {}
```

Comment:
Why is this being removed? The idea was to be able to download the camera definition in advance, because it may be slow/timeout.

Reply:
My plan is to have it initialize automatically when it discovers a camera.

Reply:
Hmm... IIRC we've had issues in the past where is was being slow and either blocking stuff or producing a lot of timeouts. Then it was lazily initialized, but it caused issues because the very first call to a camera function was sometimes super slow (again, timeouts...).

Making it explicit has a few advantages:
* If you don't use the camera at all, you don't download the camera definition
* You can download the camera definition before you start using the camera (because usually you know in advance that you will use it)

Reply:
With C++ you would actually not even instantiate the plugin but I can see that for mavsdk_server it's a different story.

Beat added file caching for MAVSDK, so if we use that to cache the FTP file, it should be fairly quick actually.

Reply:
Your call :+1:. I just wanted to mention why I added this function (in my experience the camera definition has always been the root of tons of camera problems)

Reply:
Right, I hear you. Let me try and see how it goes.